templates/s3/s3.py

```diff
@@ -15,13 +15,14 @@
 NS_TEMPLATE_UID = 'github.com/threefoldtech/0-templates/namespace/0.0.1'
 
 
+
 class S3(TemplateBase):
     version = '0.0.1'
     template_name = "s3"
 
     def __init__(self, name=None, guid=None, data=None):
         super().__init__(name=name, guid=guid, data=data)
-        self._storage_network_enabled = False
+        # self._storage_network_enabled = False
```

Comment:
Why are we not removing this all together?

Reply:
yep forgot about it. thx